splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/TableHeader.jsx

```diff
@@ -90,15 +91,15 @@ function TableHeader({ page, services, totalElement, handleRequestModalOpen }) {
             </div>
             <div>
                 <Paginator
-                    onChange={(e, { pageNumber }) => setCurrentPage(pageNumber - 1)}
+                    onChange={(e, val) => setCurrentPage(val.page - 1)}
```

Comment:
@harshpatel-crest I think this won't work. Please verify

Reply:
It works. the previous one won't work because val is javascript object and `pageNumber` is not valid property which was changed during eslint failure fixes. It should be `page`.